src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -232,6 +232,42 @@ public void testGVCFModeIsConsistentWithPastResults(final String inputFileName,
         }
     }
 
+
+    /*
+     * Minimal test that the non-seq graph haplotype detection code is equivalent using either seq graphs or kmer graphs
+     *
+     *  NOTE: --disable-sequence-graph-simplification is currently an experimental feature that does not directly match with
+     *        the regular HaplotypeCaller. Specifically the haplotype finding code does not perform correctly at complicated
+     *        sites, which is illustrated by this test. Use this mode at your own risk.
+     */
+    @Test(dataProvider="HaplotypeCallerTestInputs", enabled = false)
+    public void testGVCFModeIsConsistentWithPastResultsUsingKmerGraphs(final String inputFileName, final String referenceFileName) throws Exception {
```

Comment:
Could you replace this with a similar test that directly compares running with and without `--disable-sequence-graph-simplification` rather than comparing to past results.  It's more maintainable that way.

Reply:
I will do that, unfortunately I still cannot enable the test as the outputs still not matching in complicated sites.

Reply:
Though i'm not sure anymore. Since the past results paradigm currently exactly matches the regular seqGraph output as is, so this shouldn't make much of a difference. I'm just concerned about having to run the regular GVCF mode twice across the tests since the haplotype caller tests are already some of the longer running integration tests we have. Feel free to push back on this stance however.

Reply:
How about run it on a smaller interval?